supertokens_python/logger.py

```diff
@@ -25,11 +24,16 @@
 
 supertokens_dir = path.dirname(__file__)
 
+
+def enable_debug_logging():
+    _logger.setLevel(logging.DEBUG)
```

Comment:
where are we resetting this during each of the test in this PR?

Reply:
Added the code to set the logger to a higher level in the setup function, effectively disabling debug logs.
Also ordered the tests in a way which ensures the logger is reset every time.